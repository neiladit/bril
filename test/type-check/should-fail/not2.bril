main{

    v1: bool = const true;
    v2: int  = not v1;

}
