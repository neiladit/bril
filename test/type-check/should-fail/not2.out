line 3: assigning logic operation result to integer
logic op error!
